devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -489,6 +507,35 @@ bool HumanStateProvider::open(yarp::os::Searchable& config)
         }
     }
 
+
+    yarp::os::Bottle& fixedRotationGroup = config.findGroup("FIXED_SENSOR_ROTATION");
+    if (!fixedRotationGroup.isNull()) {
+        for (size_t i = 1; i < fixedRotationGroup.size(); ++i) {
```

Comment:
May be a debug info showing the use of `FixedRotationGroup`  config parameter will be useful

Reply:
I add a log message with the links on which secondary calibration is applied